scripts/firmware.py

```diff
@@ -1444,6 +1502,54 @@ def query_devices(self):
         menu = Menu("Options", menu_items)
         menu.display()
 
+    def enter_katapult_bootloader(self, device: str):
```

Comment:
Is this a fully new function? Should it be separate PR?

Reply:
renamed and changed a little, was needed for logging to be able to capture the outputs as it couldnt before.

Reply:
Oh yes I did move it to a seperate function to make it a little clearer. But i cbf with a seperate PR for it since its needed as apart of this PR.

Reply:
Super.